src/applications/personalization/dashboard-2/components/health-care/HealthCare.jsx

```diff
@@ -14,26 +16,35 @@ import {
 } from '~/platform/user/selectors';
 import { mhvUrl } from '~/platform/site-wide/mhv/utilities';
 import Prescriptions from './Prescriptions';
+import Appointments from './Appointments';
 import HealthCareCard from './HealthCareCard';
 
 const HealthCare = ({
   loadPrescriptions,
   prescriptions,
+  appointments,
   authenticatedWithSSOe,
   canAccessRx,
+  fetchConfirmedFutureAppointments,
   isCernerPatient,
   facilityNames,
 }) => {
   useEffect(
     () => {
+      fetchConfirmedFutureAppointments();
```

Comment:
I think this should be in its own `useEffect` with an empty dep array. I think it'd be possible for this `useEffect` to run multiple times as Redux is churning and changing state and that might trigger multiple `fetchAppt` calls.

Reply:
which leads me to think that we should try to make a habit to make our API-fetching redux actions confirm a call isn't already in flight before making an API call.

Reply:
Updated to use its own `useEffect`! 😄